02_housing/programs/Homelessness_metrics_city.do

```diff
@@ -2,20 +2,16 @@
 ** E Blom **
 ** 2020/08/04 **
 *Updated September 2022 by Emily Gutierrez
-*Updated Reb 2024 by Emily  Gutierrez
+*Updated February 2024 by Emily  Gutierrez
 *Creates the number and share of homeless students by city for 2020-21 & 2021-22 (2014-15 through 2019-20 completed in previous update)
 	*Creates the number and share of homeless by race/ethnicity for 2020-21 & 2021-22 (recreates 2019-20 with total homeless as denominator)
-
-
-
+	
 clear all
 
-global gitfolder "C:\Users\ekgut\OneDrive\Desktop\urban\Github\mobility-from-poverty"
-global years 2019 2020 2021 // refers to 2019-20 school year - most recent data
-global raceyears 2019 2020 2021 // enrollment by race data starts in 2019-20
+global gitfolder "C:\Users\ekgut\OneDrive\Desktop\urban\Github2\mobility-from-poverty"
+global years 2019 2020 2021 // refers to 2019-20 school year through most recent data
 global cityfile "${gitfolder}\geographic-crosswalks\data\place-populations.csv"
 
-
 cap n mkdir "${gitfolder}\02_housing\data"
```

Comment:
Program stops running here (it's trying to direct to C:\Users\ekgut\ but I don't have access)